css/about.css

```diff
@@ -189,7 +189,7 @@ body {
 .my-main-logo {
   border: 1px solid #d3d3d3;
   margin-top: 60px;
-  margin-bottom: 100px;  
+  margin-bottom: 100px;
```

Comment:
- [ ] One thing that is not correct in your design is in about the page background image is not covered all the background. **TIP**: Let's change the background size to cover 🎁. Please see the image below to see more information
![image](https://user-images.githubusercontent.com/69011963/152644509-f5a4c9e4-92bb-4a04-b1b8-c531307a7c2d.png)

Reply:
I managed to fix that. Thanks for the comment. 👍🎉🎉🎉